### PR TITLE
Local graph changes

### PR DESCRIPTION
**Pull request by wjwwood**
Connects to ros2/rmw_connext#168.

### CODE REVIEW

File: test_rclcpp/test/test_publisher.cpp [hunk 1]
```diff
@@ -31,52 +34,43 @@
 
 // Short test for the const reference publish signature.
 TEST(CLASSNAME(test_publisher, RMW_IMPLEMENTATION), publish_with_const_reference) {
-  rclcpp::init(0, nullptr);
-  auto node = rclcpp::Node::make_shared("test_publisher");
-
-  auto publisher = node->create_publisher<test_rclcpp::msg::UInt32>("test_publisher", 10);
-
+  // topic name
```
Comment (dirk-thomas) [inline test_rclcpp/test/test_publisher.cpp]: The comment doesn't contain more information then the following variable name.

Same below.

File: test_rclcpp/test/test_publisher.cpp [hunk 2]
```diff
@@ -31,52 +34,43 @@
...
+  std::string topic_name = "test_publish_with_const_reference";
+  // code to create the callback and subscription
```
Comment (dirk-thomas) [inline test_rclcpp/test/test_publisher.cpp]: I would suggest to remove the `code to` part in the comment.

Same multiple times below.

File: test_rclcpp/test/test_publisher.cpp (hunk 1 above)
Comment (wjwwood) [inline test_rclcpp/test/test_publisher.cpp]: 33c115d

File: test_rclcpp/test/test_publisher.cpp (hunk 2 above)
Comment (wjwwood) [inline test_rclcpp/test/test_publisher.cpp]: 33c115d

**Pull request merged**